README.md

```diff
@@ -51,14 +53,46 @@ You can download trained weights for NSRTS by following the links from the [repo
 * [https://github.com/mojivalipour/symbolicgpt](https://github.com/mojivalipour/symbolicgpt)
 * [arXiv:2106.14131](https://arxiv.org/abs/2106.14131)
 
-**coming soon**
+Clone the repo. I recommend cloning my fork:
```

Comment:
Add more instructions here in a future update. Likely this will mean adding a `setup.py` to SymGPT and updating the wrapper to use non-local imports.